packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -441,3 +446,18 @@ export function shouldPropagateTraceHeaders(
     );
   }
 }
+
+export function getURL(urlString: string | undefined) {
+  if (urlString) {
+    try {
+      return new URL(urlString);
```

Comment:
This is a `node` only type, so this won't work in a browser.

Reply:
resolved with below comment form @t2t2